flink-table/flink-table-planner/src/main/java/org/apache/calcite/avatica/util/DateTimeUtils.java

```diff
@@ -1011,6 +1103,16 @@ public static Calendar calendar() {
 		return Calendar.getInstance(UTC_ZONE, Locale.ROOT);
 	}
 
+	/** Returns whether a value is an {@code OffsetDateTime}. */
+	public static boolean isOffsetDateTime(Object o) {
+		return OFFSET_DATE_TIME_HANDLER.isOffsetDateTime(o);
+	}
+
+	/** Returns the value of a {@code OffsetDateTime} as a string. */
+	public static String offsetDateTimeValue(Object o) {
+		return OFFSET_DATE_TIME_HANDLER.stringValue(o);
+	}
```

Comment:
Does this mean if we do not have class "java.time.OffsetDateTime", we would always got an exception ?

Reply:
this is copied from avactica 1.14.0 release